react/features/filmstrip/components/web/Thumbnail.js

```diff
@@ -182,14 +278,86 @@ class Thumbnail extends Component<Props, State> {
      * @inheritdoc
      * @returns {void}
      */
-    componentDidUpdate(prevProps: Props) {
+    componentDidUpdate(prevProps: Props, prevState: State) {
         if (prevProps._audioTrack !== this.props._audioTrack) {
             this._stopListeningForAudioUpdates(prevProps._audioTrack);
             this._listenForAudioUpdates();
             this._updateAudioLevel(0);
         }
+
+        if (prevState.displayMode !== this.state.displayMode) {
+            this._onDisplayModeChanged();
+        }
+    }
+
+    /**
+     * Handles display mode changes.
+     *
+     * @returns {void}
+     */
+    _onDisplayModeChanged() {
+        const input = getDisplayModeInput(this.props, this.state);
+        const displayModeString = DISPLAY_MODE_TO_STRING[this.state.displayMode];
+        const id = this.props._participant?.id;
+
+
+        this._maybeSendScreenSharingIssueEvents(input);
+        logger.debug(`Displaying ${displayModeString} for ${id}, data: [${JSON.stringify(input)}]`);
```

Comment:
Will you keep this?

Reply:
Yeah... 

I was with mixed feelings about it. Actually I was planing to remove it but lately it has been very very useful for investigating issues. So I decided I will keep it for now.

Reply:
Ack